packages/mobile/src/screens/sign-up-screen/types.ts

```diff
@@ -0,0 +1,4 @@
+export type SignUpScreenParamList = {
+  SignUp: undefined
```

Comment:
is this still undefined?

Reply:
Yeah basically "no parameters" BUT we will have referral link there eventually

Reply:
cool 👍